node_scripts/setup_node.sh

```diff
@@ -39,7 +39,6 @@ fi
 # Unzip resource files and set permissions
 apt-get -y install unzip
 chmod 777 $AZ_BATCH_TASK_WORKING_DIR/docker_main.sh
-chmod -R 777 $AZ_BATCH_TASK_WORKING_DIR/custom-scripts
```

Comment:
why remove this? does that break custom script?

Reply:
No. this is a bug in our tool. We don't need it. I think @jafreck removed it in a separate PR but maybe not in master yet.

Reply:
Yeah, removed in #164 but still not in master yet.